tools/make/tools.mk

```diff
@@ -63,3 +63,11 @@ tools/bin/$(notdir $(SHELLCHECK_TXZ)):
 	mkdir -p $(@D)
 	tar -C $(@D) -Jxmf $< --strip-components=1 shellcheck-v$(SHELLCHECK_VERSION)/shellcheck
 endif
+
+tools.clean: # Remove all tools
+	@$(LOG_TARGET)
+	rm -rf $(tools.bindir)
+
+.PHONY: clean
+clean: ## Remove all files that are created during builds.
```

Comment:
umm there's already a `clean` target under `go.mk`

Reply:
there's another one in `docs.mk`, just follow the old way.

Reply:
ah so `make clean` will call all of them ?